packages/theme-chalk/src/cascader-panel.scss

```diff
@@ -61,8 +61,13 @@
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
-    text-align: center;
+    display: flex;
+    align-items: center;
     color: var(--el-cascader-color-empty);
+
+    .is-loading {
+      margin-right: 2px;
+    }
```

Comment:
Can we use this?

```scss

@include when(loading) {
  margin-right: 2px;
}

```

Reply:
I think the code is fine, but it doesn't seem to take effect after the modification...

Reply:
Then let's keep it this way.